docs/en/observability/ingest-splunk.asciidoc

```diff
@@ -0,0 +1,131 @@
+After {es} and {kib} are installed Fleet must be enabled, see
+https://www.elastic.co/guide/en/fleet/current/fleet-quick-start.html[Fleet Quick Start].
+
+
+=====
+
+Apache, AWS Cloudtrail, Nginx, and Zeek integrations offer the ability
+to seamlessly ingest data from a Splunk Enterprise instance.  Data
+will be automatically mapped to the Elastic Common Schema, making it
+available for rapid analysis in Elastic solutions, including Security
+and Observability.
+
+These integrations work by using the `httpjson` input in {agent} to
+run a Splunk search via the Splunk REST API and then extract the raw
+event from the results.  The raw event is then processed via the
+{agent}.  The Splunk search is customizable and the interval between
+searches is customizable.  These integrations only get new data since
+the last query, not historical data.
+
+[role="screenshot"]
+image::images/elastic-agent-splunk.png[Splunk integration components]
+
+To ingest Nginx data from Splunk the following steps are performed.
+The options are the same for Apache, AWS Cloudtrail and Zeek.
+
+== Step 1: Add integration
+
+Find the Nginx integration and begin adding it as described in
+https://www.elastic.co/guide/en/fleet/current/fleet-quick-start.html#add-nginx-integration[Fleet Guide]
+
+== Step 2: Enable Collect logs from third-party REST API
+
+Enable "Collect logs from third-party REST API" and disable both "Collect
+logs from Nginx instances" and "Collect metrics from Nginx instances".
+
+[role="screenshot"]
+image::images/kibana-fleet-third-party-rest-api.png[{fleet} showing enabling third-party REST API]
+
+== Step 3: Enter connection information
+
+Enter the required information to connect to the Splunk Enterprise REST API.
+
+The URL of the Splunk Enterprise Server must include the scheme (http or https),
+the IP address or hostname of the Splunk Enterprise Server, and the the port the
+REST API is listening on.
+
+The Splunk username and password must be of a user with a role or
+capability to use REST API endpoints.  Administrative users have these
+permissions by default.
+
+SSL Configuration is available under the "Advanced options".  These may be necessary
+if Splunk Enterprise server uses self signed certificates.  See
+https://www.elastic.co/guide/en/beats/filebeat/current/configuration-ssl.html[SSL Options]
+for valid configuration options.
+
+[role="screenshot"]
+image::images/kibana-fleet-third-party-rest-settings.png[{fleet} showing enabling third-party REST API settings]
+
+== Step 4: Enter information to select data from Splunk
+
+For each type of log file enter the interval and Splunk search string.
+
+The interval is expressed as a
+https://golang.org/pkg/time/#ParseDuration[Go duration].  The interval
+is the time between requests sent to the Splunk Enterprise REST API to
+request new information.  Intervals less than one second are not
+recommended, Splunk only maintains second accuracy for index time.
```

Comment:
This is a [comma splice](https://www.chompchomp.com/terms/commasplice.htm). A semicolon will fix that up:

```suggestion
recommended; Splunk only maintains second accuracy for index time.
```